geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/serial/SerialWANPersistenceEnabledGatewaySenderDUnitTest.java

```diff
@@ -603,27 +605,32 @@ public void testReplicatedRegionPersistentWanGateway_restartSenderWithCleanQueue
     vm7.invoke(() -> WANTestBase.createPersistentReplicatedRegion(getTestMethodName() + "_RR", "ln",
         isOffHeap()));
 
+    vm4.invoke(() -> WANTestBase.pauseSender("ln"));
+    vm5.invoke(() -> WANTestBase.pauseSender("ln"));
+
     vm4.invoke(() -> WANTestBase.doPuts(getTestMethodName() + "_RR", 1000));
 
     logger.info("Completed puts in the region");
 
     vm4.invoke(() -> WANTestBase.stopSender("ln"));
     vm5.invoke(() -> WANTestBase.stopSender("ln"));
 
-    logger.info("Stopped all the senders. ");
 
-    // Create receiver on remote site
-    createReceiverInVMs(vm2, vm3);
+    logger.info("Stopped all the senders. ");
```

Comment:
Instead of using the logger in this test to track progress, you could instead use the `invoke()` method signature that passes a String description/name as the first parameter, which allows anyone debugging the test to see when the invocation in question starts and finishes. Just personal preference, but I think it makes things neater and provides a little more information to anyone looking at the test output.

Reply:
A valid point Donal. I am creating another GEODE ticket to just remove all the logger.info in tests. There are a couple of more statements that went in other commits.